From: Jon Bernard <jbernard@debian.org>
Date: Wed, 15 Dec 2010 10:24:29 -0500
Subject: [PATCH] Remove missing file reference from cgred initscript
To: libcg-devel@lists.sourceforge.net

The file /etc/rc.d/init.d/functions does not exist in Debian. Sourcing
only /lib/lsb/init-functions should be all that is required.
---
 scripts/init.d/cgred.in |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/scripts/init.d/cgred.in b/scripts/init.d/cgred.in
index 936597c..4b6c610 100644
--- a/scripts/init.d/cgred.in
+++ b/scripts/init.d/cgred.in
@@ -40,7 +40,6 @@ CGRED_CONF=/etc/cgrules.conf
 [ -x $CGRED_BIN ] || exit 1
 
 # Source function library & LSB routines
-. /etc/rc.d/init.d/functions
 . /lib/lsb/init-functions
 
 # Read in configuration options.
-- 
